kotsu/store.py

```diff
@@ -0,0 +1,19 @@
+"""Functionality for storing validation results."""
+from typing import List
+
+import os.path
+
+import pandas as pd
+
+
+def write(results_list: List[dict], store_directory: str, to_front_cols: List[str]):
+    """Write the results to the store directory."""
+    df = pd.DataFrame(results_list)
```

Comment:
i think this is the only place we use `pandas` - I think it's probably worth considering using the stdlib CSV writer here to avoid the v heavyweight pandas dependency and all the compiled C etc etc that comes with it (but one for a diff PR)

Reply:
Thought added to #19